twingate/resource_resource_test.go

```diff
@@ -602,7 +616,8 @@ func TestAccTwingateResourceSetActiveStateOnUpdate(t *testing.T) {
 			},
 			{
 				Config: createResource10(remoteNetworkName, resourceName),
-				Check: resource.ComposeTestCheckFunc(
+				Check: ComposeTestCheckFunc(
+					wait(),
```

Comment:
This is a mistake... you already wait at the beginning in `ComposeTestCheckFunc`

Reply:
agree